code/modules/unit_tests/slips.dm

```diff
@@ -0,0 +1,13 @@
+/datum/unit_test/aaa/Run()
```

Comment:
actually name it something silly

also split it like this:

```
/// Doc comment explaining what the test does here
/datum/unit_test/test_name

/datum/unit_test/test_name/Run()

Reply:
Aaaa I forgot to rename it. It was just meant to be `slips`. Changed.